joomla/joomla-cms
# Debug permissions in com_users

We have a very nice and helpful view for "debugging" issues with user permissions. But that view is only available when "Debug System" is enabled in the global configuration. If it's disabled (like in normal production) the link to it is hidden.

### Summary of Changes
This PR adds two new options in the user manager which will allow to enable/disable that ~"Debug"~ "Advanced" view for the user and usergroup manager. Making the view more known.
The new options are enabled by default since there is no drawback to have them enabled. It's just a link shown. Also it will be B/C for those who had Debug enabled for whatever reason.

### Testing Instructions
* Apply the PR and test the new links right below each user/usergroup.
* Disable the option in the component options and make sure the link no longer appears.

### Expected result
Have a link to those views not depending on Debug status.

### Actual result
Links only shown when Debug is active.

### Documentation Changes Required
Maybe tutorials need to be adjusted.

#### Disclaimer
Work is done by @brianteeman
